lib/Doctrine/ORM/Tools/EntityGenerator.php

```diff
@@ -277,10 +277,12 @@ public function <methodName>(<methodTypeHint>$<variableName>)
  * <description>
  *
  * @param <variableType> $<variableName>
+ *
+ * @return boolean TRUE if this collection contained the specified element, FALSE otherwise.
```

Comment:
I don't care tbh, was just nitpicking because there was a PR about this kinda thing recently...